app/src/main/java/deakin/gopher/guardian/model/MedicalDiagnostic.kt

```diff
@@ -0,0 +1,22 @@
+package deakin.gopher.guardian.model
+
+data class MedicalDiagnostic(
+    var patientId: String? = null,
```

Comment:
I'd say patientId, name and current shouldn't be nullable. They should be required for all patient's.